src/content/en/fundamentals/performance/lazy-loading-guidance/images-and-video/index.md

```diff
@@ -0,0 +1,699 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: If your site has a ton of images and video, but you don't want to cut down on any of it, lazy loading might be just the technique you need to improve initial page load time and lower per-page payload.
+
+{# wf_updated_on: 2018-03-23 #}
+{# wf_published_on: 2018-04-04 #}
+{# wf_blink_components: Blink>PerformanceAPIs,Blink>JavaScript>Runtime,Blink>Input #}
+
+# Lazy Loading Images and Video {: .page-title }
+
+{% include "web/_shared/contributors/jeremywagner.html" %}
+
+The portion of [images](http://beta.httparchive.org/reports/state-of-images?start=earliest&end=latest) and [video](http://beta.httparchive.org/reports/page-weight#bytesVideo) in per-page payloads of typical websites can be significant. Unfortunately, project stakeholders may be unwilling to cut any media resources from their existing applications. Such impasses are frustrating, especially when all parties involved want to improve site performance, but can't agree on how to get there. Fortunately, lazy loading is a solution that lowers initial page payload _and_ load time, but doesn't skimp on content.
```

Comment:
I'd consider simplifying or rewording the opening sentence (hard to parse on re-reading). Perhaps, the portion of images and video in the typical payload of a website can be significant?